oss-licenses-plugin/src/main/groovy/com/google/android/gms/oss/licenses/plugin/DependencyUtil.groovy

```diff
@@ -146,7 +146,7 @@ class DependencyUtil {
                         }
                     }
                 }
-            } catch (AmbiguousVariantSelectionException exception) {
+            } catch (ArtifactSelectionException exception) {
```

Comment:
@tresat on older versions of Gradle running this branch may throw an exception when JVM tries to find the `catch` block for `AmbiguousVariantSelectionException` when it is thrown, because this new `ArtifactSelectionException` might not be available in those old versions of Gradle. (I have no proof, just something that came to mind when seeing this change.)

Reply:
Definitely something to test.